sandbox/build.gradle

```diff
@@ -63,8 +63,10 @@ dependencies {
     implementation deps.external.mviCoreAndroid
     implementation deps.external.rxrelay2
     implementation deps.external.rxandroid2
+    implementation deps.external.leakCanary // use debugImplementation in real life apps
 
     implementation project(":libraries:rib-base")
+    implementation project(":libraries:rib-debug-utils")
```

Comment:
maybe we should include debug utils only for debug build? I think we can create a class that will provide debug plugins in debug source set and no plugins in release source set

Reply:
#217